FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift

```diff
@@ -118,21 +120,40 @@ extension ThreadDetailsViewController {
     }
 
     private func handleReplyTap(at indexPath: IndexPath) {
+        composeNewMessage(at: indexPath, quoteType: .reply)
+    }
+
+    private func handleForwardTap(at indexPath: IndexPath) {
+        composeNewMessage(at: indexPath, quoteType: .forward)
+    }
+
+    private func composeNewMessage(at indexPath: IndexPath, quoteType: MessageQuoteType) {
         guard let email = DataService.shared.email,
               let input = input[safe: indexPath.section-1],
               let processedMessage = input.processedMessage
         else { return }
 
-        let replyInfo = ComposeMessageInput.ReplyInfo(
-            recipient: input.rawMessage.sender,
-            subject: input.rawMessage.subject,
+        let recipient: String?
+        switch quoteType {
+        case .reply:
+            recipient = input.rawMessage.sender
+        case .forward:
+            recipient = nil
+        }
```

Comment:
Might as well change the recipient to an array, as will be needed for replyAll anyway. #953 . Then instead of nil, it will be empty array?

Reply:
Yeah, having array here will be more future-proof, I'll update code

Reply:
Should I also add UI test for messages forwarding or it'll be separate issue?

Reply:
Yes - please add UI test that opens encrypted message and taps forward. Then evaluate that the quote is present, and that there are no recipients. And the subject. That would end the test successfully (tests that don't change state are easier for now, until we test against mock of gmail api)